site_scons/site_tools/scons_ninja/Globals.py

```diff
@@ -0,0 +1,51 @@
+# MIT License
+#
+# Copyright The SCons Foundation
```

Comment:
I'm a bit picky about the folder structure of the Godot repository, and I like the top-level folders to be all Godot code or documentation - buildsystem, CI, and packaging tooling go to `misc` usually.

Could you put this in `misc/scons/scons_ninja/` using the `toolpath` argument in `env.Tool()`? As we used to do in #38830 before the compilation db was upstreamed.

Same goes for other PRs adding tools to SCons' default `site_scons/site_tools/`.

Reply:
done in 
e3b3d25